native/src/utils/AppSettings.ts

```diff
@@ -168,6 +168,6 @@ class AppSettings {
   }
 }
 
-export const appSettings = new AppSettings()
+const appSettings = new AppSettings()
 
-export default AppSettings
+export default appSettings
```

Comment:
Just merging the two lines into one should be fine.

Reply:
I prefer having named constants instead of just exporting something as default :)